profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -82,12 +82,16 @@ function nexteuropa_varnish_cache_purge_rule_label($purge_rule) {
  * Implements hook_workbench_moderation_transition().
  */
 function nexteuropa_varnish_workbench_moderation_transition($node, $previous_state, $new_state) {
-  $purge
-    = ($new_state == workbench_moderation_state_published()) ||
-      (property_exists($node->workbench_moderation['my_revision'], 'unpublishing')
-        && $node->workbench_moderation['my_revision']->unpublishing);
+  if (isset($node->is_draft_revision) ? $node->is_draft_revision : FALSE) {
+    return;
+  }
+
+  $publishing = $new_state == workbench_moderation_state_published();
+
+  $unpublishing = (($new_state != workbench_moderation_state_published()) &&
```

Comment:
We could replace `$new_state != workbench_moderation_state_published()` with `!$publishing`.